package.json

```diff
@@ -2,14 +2,17 @@
   "name": "persistent-tunnel",
   "version": "1.2.4",
   "description": "HTTP Agent for tunneling proxies with persistent sockets",
-  "main": "index.js",
+  "main": "lib/index.js",
+  "typing": "lib/index.d.ts",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "pretest": "./node_modules/.bin/coffee -o build -c lib",
-    "test": "./node_modules/.bin/mocha --compilers coffee:coffee-script/register",
-    "prepublish": "./node_modules/.bin/coffee -o build -c lib"
```

Comment:
It would be good to add `require-npm4-to-publish` in order to make sure this repo isn't accidentally published by an old npm that ignores the `prepare` script

Reply:
Fixed